components/footer/FirstBlock.tsx

```diff
@@ -23,13 +23,13 @@ export const FirstBlock = (
       </div>
       <div class="flex flex-row justify-between  items-center w-full max-w-[240px] lg:max-w-sm ">
         <a href={instagram}>
-          <Icon width="50px" height="50px" id="Instagram" />
+          <Icon width="50px" height="50px" id="Instagram" style={{ color: "#000000" }} />
```

Comment:
Com o currentColor no svg voce so precisa colocar a cor via classes mesmo, tipo:
```suggestion
        <a class="text-black">
          <Icon width="50px" height="50px" id="Instagram" />
        </a>
```